manifests/n/Nethesis/NethLink/1.0.2/Nethesis.NethLink.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Nethesis.NethLink
+PackageVersion: 1.0.2
+PackageLocale: en-US
+Publisher: NethesisSRL
+PackageName: NethLink
+License: MIT
+Copyright: copyright

```

Comment:
Please provide the full copyright information including the year of the copyright and the attribution